src/App.tsx

```diff
@@ -1,5 +1,56 @@
+const colors = [
+    {
+        hex: 'EFDECD',
+        name: 'Almond',
+        comp: [
+            { hex: 'ADC6E0', name: 'Periwinkle Gray' },
+            { hex: 'D8E6F5', name: 'Link Water' },
+            { hex: 'FFFFFF', name: 'White' },
+        ],
+    },
+    {
+        hex: 'CD9575',
+        name: 'Antique Brass',
+        comp: [
+            { hex: '649DBE', name: 'Cerulean Frost' },
+            { hex: '88BCDA', name: 'Half Baked' },
+            { hex: 'EFC9B3', name: 'Desert Sand' },
+        ],
+    },
+];
+
 const App = () => {
-    return <h1>Hello World</h1>;
+    return (
```

Comment:
the App component, and its return statement, is doing too much here. You could consider breaking out some of the logic below into sub-components ( PrimaryColor, ComplementaryColor) to improve readability.